components/molecule/thumbnail/src/index.scss

```diff
@@ -0,0 +1,97 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$sizes-sui-molecule-thumbnail: (
+  large: 144px,
+  medium: 72px,
+  small: 48px,
+  xsmall:  40px
+) !default;
+
+$ratios-sui-molecule-thumbnail: (
+  1: 1,
+  4: 3,
+  16: 9
+) !default;
+
+$bd-outer-sui-molecule-thumbnail: 1px solid color-variation($c-gray, 3) !default;
+$bdc-outer-hover-sui-molecule-thumbnail: $c-text-link !default;
```

Comment:
What about using: `$bdc-molecule-thumbnail-outer-hover`

instead of `$bdc-outer-hover-sui-molecule-thumbnail` ?

i think it fits better with our convention

Reply:
sure